app/src/utils/inputSanitizer.ts

```
@@ -0,0 +1,345 @@
+          return null;
+        }
+        sanitized[key] = sanitizeString(value, {
+          maxLength: rules.maxLength
+        });
+        break;
+
+      case 'number':
+        const numValue = sanitizeNumber(value, {
+          min: rules.min,
+          max: rules.max
+        });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)